content/scalers/apache-kafka-topic.md

```diff
@@ -5,7 +5,7 @@ title = "Apache Kafka Topic"
 background = "light"
 +++
 
-Scale applications based on Apache Kafka Topic.
+Scale applications based on Apache Kafka Topic. This can also be used to scale applications based on Azure Event Hubs w/Kafka enabled.
```

Comment:
To be honest I don't think that this kind of "specific" use case should be mentioned. Who is using AEH with Kafka head knows that can use this trigger because ... it's about Kafka topic.

Reply:
That's a fair point, what about this instead @ppatierno?
```suggestion
Scale applications based on Apache Kafka Topic or other services that support Kafka protocol.
```
Or:
```suggestion
Scale applications based on Apache Kafka Topic or other services that support Kafka protocol such as Azure Event Hubs.
```

Reply:
Thanks for the feedback!
I definitely like the 2nd recommendation made by @tomkerkhove

Reply:
I am more for the first one, because if at some point another service will come supporting the Kafka protocol, then we should mention it here as well. For this reason, I would be more generic.
Then, of course, having a dedicated example of using Azure EH (as I mentioned in another comment).